### ISSUE TITLE
Geometric sample with ANGLE: black window

### ISSUE THREAD

**Issue by viric** [cinder/Cinder#1734]
I have been trying to run the Geometric sample with ANGLE x64 on Windows, and I can't make it work. I only get a window completely black. I tried the 'Debug' target.

I had to fix the build recipe for Angle/Debug in the sample, as it lacked x64 and Debug_ANGLE targets (that included the angle flags).

Why is it giving just a black window?

**Comment by paulhoux** [comment]
Hi,

because of the huge number of targets Cinder is currently supporting (Windows, Mac, Linux, Android, each with their own set of back-ends), the team has not yet been able to setup automated testing for all platforms. Therefor, not all samples support all targets and platforms yet. The `Geometry` sample is one of them.

However, you can manually make a few changes to the project to make it work with ANGLE. See [this post](https://forum.libcinder.org/topic/glnext-es-3-angle) for more information. I've checked and can confirm the steps mentioned in the post are still correct.

**Comment by viric** [comment]
I could build it with ANGLE without trouble (doing changes like those you mention). It just gives me a black window when I run it.

Have you checked whether it paints fine for you?

**Comment by paulhoux** [comment]
Yes, it works fine. I can tell it's using ANGLE because the Params window is disabled (it's not compatible with ANGLE). Lines are not anti-aliased, which is also an indication it's running on ANGLE. Other than that, the sample works fine and I can switch between primitives using the space bar.

**Comment by viric** [comment]
Interesting. Whatever I try, with ANGLE with this sample, I only get a black window. Other samples work fine for me.
